org.alloytools.alloy.classic.provider/src/main/java/org/alloytools/alloy/classic/solver/kodkod/UnsatKodkodOptions.java

```diff
@@ -0,0 +1,11 @@
+package org.alloytools.alloy.classic.solver.kodkod;
+
+@Description("Options for the Unsat facility of Kodkod")
+public class UnsatKodkodOptions extends KodkodOptions {
+
+	@Description("Unsat core granularity, " + "default is 0 (only top-level conjuncts are considered), "
```

Comment:
what's the point of this string concatenation when both strings are on the same line?

Reply:
formatter